src/Reflection/Adapter/ReflectionObject.php

```diff
@@ -458,7 +428,9 @@ public function isSubclassOf($class)
             return strtolower($parentClassName);
         }, $realParentClassNames), $realParentClassNames);
 
-        $realParentClassName = $parentClassNames[strtolower($class)] ?? $class;
+        $lowercasedClass = strtolower($class);
```

Comment:
Similar to above: what happened here, that required extracting to variable? (mostly curious, not important)